src/components/Dashboard/ThisWeek/NewsletterMenu/NewsletterMenu.tsx

```diff
@@ -0,0 +1,303 @@
+import React, { ReactElement, useState, useEffect } from 'react';
+import {
+  Button,
+  Menu,
+  MenuItem,
+  Dialog,
+  DialogTitle,
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+  ListItemText,
+  IconButton,
+  styled,
+  TextareaAutosize,
+} from '@material-ui/core';
+import ArrowDropDown from '@material-ui/icons/ArrowDropDown';
+import CloseIcon from '@material-ui/icons/Close';
+import { useTranslation } from 'react-i18next';
+import { DateTime } from 'luxon';
+import { Skeleton } from '@material-ui/lab';
+import {
+  useGetTaskAnalyticsQuery,
+  useGetEmailNewsletterContactsLazyQuery,
+} from '../NewsletterMenu.generated';
+
+interface Props {
+  accountListId: string;
+}
+
+const CloseButton = styled(IconButton)(({ theme }) => ({
+  position: 'absolute',
+  right: theme.spacing(1),
+  top: theme.spacing(1),
+  color: theme.palette.grey[500],
```

Comment:
@TheNoodleMoose I'd change this gray tone to `theme.palette.text.primary`. I'm not sure where it's defined, but the light gray tone used for the background circle on hover should be `#EBECEC`. That light gray shade isn't available in the theme yet, but if you hard code it, I'll fix it as I'm working on the theme overhaul.